python/pyspark/sql/dataframe.py

```diff
@@ -1422,6 +1422,53 @@ def colRegex(self, colName: str) -> Column:
         jc = self._jdf.colRegex(colName)
         return Column(jc)
 
+    def convert(self, schema: StructType) -> "DataFrame":
+        """
+        Returns a new DataFrame where each row is reconciled to match the specified schema.
```

Comment:
The first line of docstring should have the summary. After that, a newline with detailed explanation